source/wifi/wifi_hal.c

```diff
@@ -8604,13 +8603,13 @@ INT wifi_getRadioVapInfoMap(wifi_radio_index_t index, wifi_vap_info_map_t *map)
             return RETURN_ERR;
         }
 
-        STRSCPY(map->vap_array[i].bridge_name, "brlan0");
+        strncpy(map->vap_array[i].bridge_name, "brlan0", sizeof("brlan0") - 1);
```

Comment:
you need to ensure strncpy doesn't try to copy more bytes that 'destination' buffer has. In this case string "brlan0" is probably shorted than bridge_name size but it's not correct to assume that. use "sizeof(map->vap_array[i].bridge_name) -1" instead.

Reply:
done